docs/uml/main/ExtEvSimulationClasses.puml

```diff
@@ -1,223 +1,202 @@
-        + UUID getUuid()
-        + String getId()
-        + ComparableQuantity<Power> getSRated()
-        + ComparableQuantity<Energy> getEStorage()
-        + ComparableQuantity<Energy> getStoredEnergy()
-        + EvModel copyWith(ComparableQuantity<Energy> newStoredEnergy)
+        class RequestCurrentPrices
+
+        class RequestDepartingEvs
+
+        class RequestEvcsFreeLots
     }
 
-    ExtDataSimulation -[hidden]> Arrival
-    EvMovement *-> EvModel
-    EvModelImpl --|> EvModel
+    RequestDepartingEvs --u|> EvDataMessageFromExt
+    RequestEvcsFreeLots --u|> EvDataMessageFromExt
+    ProvideArrivingEvs --u|> EvDataMessageFromExt
+    RequestCurrentPrices --u|> EvDataMessageFromExt
 
+    DataMessageFromExt --d|> EvDataMessageFromExt
+    ExtEvData --r[norank]> EvDataMessageFromExt
+    ExtEvData --u|> ExtData
+    ExtEvSimulation -u> ExtEvData
 
-    interface EvDataMessageFromExt
-    class EvMovementsMessage {
-        - Map<UUID, List<EvMovement>> movements
-    }
-    class RequestEvcsFreeLots
-    RequestEvcsFreeLots --|> EvDataMessageFromExt
-    EvMovementsMessage --|> EvDataMessageFromExt
-    RequestEvcsFreeLots -[hidden]> EvMovementsMessage
-
-    interface EvDataResponseMessageToExt
-    class AllDepartedEvsRepsonse {
-        - Map<UUID, List<EvMovement>> movements
-    }
-    class ProvideEvcsFreeLots
-    ProvideEvcsFreeLots --|> EvDataResponseMessageToExt
-    AllDepartedEvsRepsonse --|> EvDataResponseMessageToExt
-    ExtEvData -> EvDataMessageFromExt
-    ExtEvData -> EvDataResponseMessageToExt
-    EvMovement -[hidden]-> RequestEvcsFreeLots
-    EvDataMessageFromExt -[hidden]> EvDataResponseMessageToExt
-    EvMovementsMessage -[hidden]> ProvideEvcsFreeLots
-    ProvideEvcsFreeLots -[hidden]> AllDepartedEvsRepsonse
-
-    class ScheduleDataServiceMessage {
-        - ExtEvDataService dataService
-    }
-    EvDataResponseMessageToExt -[hidden]> ScheduleDataServiceMessage
-    ExtEvData -> ScheduleDataServiceMessage
+    node sub1{
+        interface DataResponseMessageToExt
 
+        interface EvDataResponseMessageToExt
 
-    ' LEFT PART
-    interface ExtLinkInterface {
-        + ExtSimulation getExtSimulation()
-        + List<ExtDataSimulation> getExtDataSimulations()
-    }
-    ExtLinkInterface -[hidden]> ExtSimulation
-    ExtLink --|> ExtLinkInterface
+        class ProvideCurrentPrices{
+            + ProvideCurrentPrices()
+        }
+
+        class ProvideDepartingEvs{
+            + ProvideDepartingEvs()
+        }
 
-    interface ExtTrigger
-    class ActivationMessage {
-        - Long tick
+        class ProvideEvcsFreeLots{
+             + ProvideEvcsFreeLots()
+        }
+
+         ProvideEvcsFreeLots --|> EvDataResponseMessageToExt
+         ProvideDepartingEvs --|> EvDataResponseMessageToExt
+         ProvideCurrentPrices --|> EvDataResponseMessageToExt
+         ExtEvData --> EvDataResponseMessageToExt
+         DataResponseMessageToExt --|> EvDataResponseMessageToExt
```

Comment:
This arrow needs to be the other way around. Maybe you can separate `DataResponseMessageToExt` from the other classes on the left as well.